sdk/batch/batch/test/batchClient.spec.ts

```diff
@@ -774,6 +781,7 @@ describe("Batch Service", () => {
       assert.equal(result._response.status, 204);
     });
 
+    // This hangs
```

Comment:
Does this test still hang? Should we log a follow up issue for this?

Reply:
Yes it hangs using my batch account. I would suggest the batch team double check this before creating an issue. cc @xingwu1.

Reply:
After calling wait, this test succeeds now.

Reply:
Then, lets remove the comment

```suggestion
```

Reply:
@ramya-rao-a fixed here https://github.com/Azure/azure-sdk-for-js/pull/11836/commits/ec851affbe06dccd13504b222b02d4cb8c07fd7a.